content/en/logs/log_configuration/rehydrating.md

```diff
@@ -13,7 +13,7 @@ Log Rehydration* enables you to capture log events from customer-owned storage-o
 
 ## Historical views
 
-With historical views, teams rehydrate archived log events precisely by timeframe and query filter to meet specific, unexpected use cases efficiently. 
+With historical views, teams rehydrate archived log events precisely by timeframe and query filter to meet specific, unexpected use cases efficiently.
```

Comment:
```suggestion
With historical views, teams can rehydrate archived log events by timeframe and query filter for specific use cases that come up unexpectedly.
```

Reply:
I'm not on the logs team, so i'd prefer not to edit their wording as a driveby change, but we could if you really want